Cargo.toml

```diff
@@ -132,6 +132,17 @@ jj-lib = { path = "lib", version = "0.22.0" }
 jj-lib-proc-macros = { path = "lib/proc-macros", version = "0.22.0" }
 testutils = { path = "lib/testutils" }
 
+[workspace.lints.clippy]
+all = "warn"
```

Comment:
this appears to demote default-denied lints to warning.

Reply:
You're right, moreover this is not needed anymore since I want to let them in the default state.